packages/luna/src/components/PlanCard/PlanCard.tsx

```diff
@@ -0,0 +1,178 @@
+import { Typography } from "@/components/Typography";
+import { useFormatMessage } from "@/hooks";
+import cn from "classnames";
+import React from "react";
+import { formatNumber } from "@/components/utils";
+
+type Props = {
+  /**
+   * the title of the plan wheather
+   *  - beginner
+   *  - advanced
+   *  - professional
+   */
+  title: string;
+  /**
+   * sammary about the plan
+   */
+  description: string;
+  /**
+   * number of minutes per week
+   */
+  weeklyMinutes: number;
+  /**
+   * price in egp (no need to scale or unscale)
+   */
+  price: number;
+  label?: "most-common" | "most-valuable";
+  primary?: boolean;
+  /**
+   * floating number between 0 and 100.
+   */
+  discount?: number;
+  onBuy(): void;
+};
+
+export const PlanCard: React.FC<Props> = ({
+  title,
+  description,
+  weeklyMinutes,
+  price,
+  label,
+  primary,
+  onBuy,
+  discount,
+}) => {
+  const intl = useFormatMessage();
+  const priceAfterDiscount = discount
+    ? price - (price * discount) / 100
+    : price;
+
+  return (
+    <div
+      className={cn(
+        "tw-relative tw-p-8 tw-border tw-border-natural-200 tw-shadow-feedback-card tw-rounded-3xl",
```

Comment:
the `tw-shadow-feeback-card` is irrelevant here. Please use a suitable classname.